content/en/docs/instrumentation/java/libraries.md

```diff
@@ -23,3 +23,189 @@ some instrumentation libraries for your dependencies standalone. To find out
+
+/**
+ * All SDK management takes place here, away from the instrumentation code, which should only access
+ * the OpenTelemetry APIs.
+ */
+class ExampleConfiguration {
+
+  /**
+   * Initializes the OpenTelemetry SDK with a logging span exporter and the W3C Trace Context
+   * propagator.
+   *
+   * @return A ready-to-use {@link OpenTelemetry} instance.
+   */
+  static OpenTelemetry initOpenTelemetry() {
+    SdkTracerProvider sdkTracerProvider =
+        SdkTracerProvider.builder()
+            .addSpanProcessor(SimpleSpanProcessor.create(new LoggingSpanExporter()))
+            .build();
+
+    OpenTelemetrySdk sdk =
+        OpenTelemetrySdk.builder()
+            .setTracerProvider(sdkTracerProvider)
+            .setPropagators(ContextPropagators.create(W3CTraceContextPropagator.getInstance()))
+            .build();
+
+    Runtime.getRuntime().addShutdownHook(new Thread(sdkTracerProvider::close));
+    return sdk;
+  }
+}
+```
+
+```java
+// SampleHttpClient.java
+package otel;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.httpclient.JavaHttpClientTelemetry;
+import java.net.http.HttpClient;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpHeaders;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+public final class SampleHttpClient {
+    //Init OpenTelemetry
+    private static final OpenTelemetry openTelemetry = ExampleConfiguration.initOpenTelemetry();
+
+    //Use this HttpClient implementation for making standard http client calls.
+    public HttpClient createTracedClient(OpenTelemetry openTelemetry) {
+        return JavaHttpClientTelemetry.builder(openTelemetry).build().newHttpClient(createClient());
+    }
+
+    //your configuration of the Java HTTP Client goes here:
+    private HttpClient createClient() {
+        return HttpClient.newBuilder().build();
+    }
+
+    public static void main(String[] args) throws Exception {
+        HttpRequest request = HttpRequest.newBuilder()
+            .GET()
+            .uri(URI.create(System.getenv().getOrDefault("EXTERNAL_API_ENDPOINT", "http://localhost:8080/rolldice")))
+            //.setHeader("User-Agent", "Java 11 HttpClient Bot") // add request header
+            .build();
+
+        SampleHttpClient s = new SampleHttpClient();
+        HttpResponse<String> response = s.createTracedClient(openTelemetry).send(request, HttpResponse.BodyHandlers.ofString());
+        // print response headers
+        HttpHeaders headers = response.headers();
+        headers.map().forEach((k, v) -> System.out.println(k + ":" + v));
+        // print status code
+        System.out.println(response.statusCode());
+        // print response body
+        System.out.println(response.body());
+
+    }
+}
+```
+
+```sh
+# .env
+export OTEL_SERVICE_NAME=dice-client
+export OTEL_TRACES_EXPORTER=logging
+export OTEL_METRICS_EXPORTER=logging
+export OTEL_LOGS_EXPORTER=logging
```

Comment:
these are not used

Reply:
Let me check and remove if not required.

Reply:
I have fixed this.